### ISSUE TITLE
Excessive wakeups and playback stalled notification

### ISSUE THREAD

**Issue by trustypelletgun** [JohnCoates/Aerial#56]
Below is an example of the set of console entries I'm seeing each time the screensaver is active for more than ~3 minutes (i.e. long enough for the first video to end and a second to begin). There is a pause after the first couple of frames of the second video play before it resumes playing the second video.

```
10/28/15 3:14:31.342 PM ScreenSaverEngine[3677]: ### archList:(16777223) --- 0
10/28/15 3:17:48.532 PM ScreenSaverEngine[3677]: AVPlayerItemPlaybackStalledNotification NSConcreteNotification 0x7facb24b5150 {name = AVPlayerItemPlaybackStalledNotification; object = <AVPlayerItem: 0x7facb492b9e0, asset = <AVURLAsset: 0x7facb2565820, URL = http://a1.phobos.apple.com/us/r1000/000/Features/atv/AutumnResources/videos/b6-3.mov>>}
10/28/15 3:18:02.000 PM kernel[0]: process ScreenSaverEngin[3677] caught causing excessive wakeups. Observed wakeups rate (per sec): 213; Maximum permitted wakeups rate (per sec): 150; Observation period: 300 seconds; Task lifetime number of wakeups: 45019
10/28/15 3:18:02.127 PM com.apple.xpc.launchd[1]: (com.apple.ReportCrash[3742]) Endpoint has been activated through legacy launch(3) APIs. Please switch to XPC or bootstrap_check_in(): com.apple.ReportCrash
10/28/15 3:18:02.129 PM ReportCrash[3742]: Invoking spindump for pid=3677 wakeups_rate=213 duration=212 because of excessive wakeups
10/28/15 3:18:02.881 PM spindump[568]: Saved wakeups_resource.diag report for ScreenSaverEngine version 5.0 (5.0) to /Library/Logs/DiagnosticReports/ScreenSaverEngine_2015-10-28-151802_mac0101680.wakeups_resource.diag
```

Hardware Overview:

  Model Name:   MacBook Pro
  Model Identifier: MacBookPro12,1
  Processor Name:   Intel Core i7
  Processor Speed:  3.1 GHz
  Number of Processors: 1
  Total Number of Cores:    2
  L2 Cache (per Core):  256 KB
  L3 Cache: 4 MB
  Memory:   16 GB
  Boot ROM Version: MBP121.0167.B07
  SMC Version (system): 2.28f7

**Comment by JohnCoates** [comment]
Try this caching version please https://github.com/JohnCoates/Aerial/releases/tag/v1.2beta

**Comment by trustypelletgun** [comment]
I am still seeing the same entries and crash logs in the system.log when using Aerial. I have continued to see this behavior while using each successive release including the current master. Caching is enabled and all videos have been downloaded. I have verified that I do not see these entries when using other screensavers.

Since this is not causing any freezing or crashing, it does not appear to be a significant issue. However, it does create excessive log files and most likely increases power consumption while using Aerial. I will continue to investigate.
